src/views/SelectProduct.vue

```diff
@@ -250,16 +234,58 @@ export default defineComponent({
             "q.op": "AND"
           } as any,
           "query": "*:*",
-          "filter": "docType: PRODUCT"
+          "filter": ["docType: PRODUCT"]
         }
+      } as any
+    }
+  },
+  methods: {
+    updateInclusionQuery(value: string, type: string) {
+      this.included[type].includes(value) ? this.included[type].splice(this.included[type].indexOf(value), 1) : this.included[type].push(value)
```

Comment:
Assign this.included[type] to a variable and then use

Reply:
Done Sir